books/kestrel/crypto/r1cs/sparse/p1cs.lisp

```diff
@@ -23,6 +23,8 @@
 ;; If you don't do this you get a program-mode-only str::pretty
 (include-book "std/strings/pretty" :dir :system)
 
+(include-book "kestrel/alists-light/top" :dir :system)
```

Comment:
I would make this local. I.e., `(local (include-book ...))`.

Local events are used to certify a book, but not "exported" to anyone including this book. See `:doc local`.